src/Features/Core/Portable/Completion/CompletionService.cs

```diff
@@ -162,7 +162,12 @@ public virtual TextSpan GetDefaultCompletionListSpan(SourceText text, int caretP
             Document document,
             CompletionItem item,
             CancellationToken cancellationToken = default)
-            => GetDescriptionAsync(document, item, CompletionOptions.From(document.Project), SymbolDescriptionOptions.From(document.Project), cancellationToken);
+        {
+            Debug.Fail("For backwards API compat only, should not be called");
```

Comment:
i do worry about some public consumer being compiled in debug mode, and now they keep hitting this assert while testing, and they can't call the internal method in the meantime.

Reply:
That would only happen if they compiled their Roslyn binaries in debug from source. We do not release debug builds of Roslyn in nuget packages.

Reply:
ah, i didn't know that.  thanks!  i'm fine with this then.